.github/workflows/ci.yml

```diff
@@ -0,0 +1,190 @@
+name: WebRTC C SDK CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  clang-format-check:
+    runs-on: macos-11
```

Comment:
The clang-format check in the travis script was on Linux, any specific reason why we moved it to MacOS here?

Reply:
Technically, it can be run on anything. The clang-format build fails on Travis. The clang-format version being used is currently not available on Ubuntu (linux provides an older version), so using Mac.